xlson/__init__.py

```diff
@@ -54,6 +55,19 @@ def __init__(self, **kwargs: Dict) -> None:
         elements[KEY] = kwargs[NAME]
         elements[TYPE] = self.field_type
 
+        if CHILDREN in elements:
```

Comment:
Can this check be done in `NativeRadioField` instead? Some native fields may have `children` but the children could be as a result of a group function and not necessarily a `select one` or `select multiple` question. When we implement the spinner function, the structure will be different.